src/mocks/streamers.ts

```diff
@@ -0,0 +1,67 @@
+import { StreamerResponse } from "@types";
+
+export const mockStreamers: Record<string, StreamerResponse> = {
+  streamer1: {
+    youtube: {
+      id: "UC1234567890",
+      name: "Sample Streamer 1",
+      icon: "https://via.placeholder.com/150/FF6B6B/FFFFFF?text=S1",
```

Comment:
You can use real thumbnail. Use thumbnail url of user https://youtube.com/@uruhaichinose?si=nRBKVLVORthBxgw5 
Also update for all other streamers